apps/design-system/src/pages/view-preview/root-view-wrapper.tsx

```diff
@@ -1,33 +1,43 @@
 import { FC, PropsWithChildren } from 'react'
 
-import { Breadcrumb, Topbar } from '@harnessio/ui/components'
-
 import { AppViewWrapper, AppViewWrapperProps } from './app-view-wrapper'
 
+export const mockedBreadcrumbs = [
+  {
+    data: undefined,
+    handle: {
+      asLink: true,
+      breadcrumb: () => <span>Lorem</span>
```

Comment:
Do we need these `span`s? They don't seem to do anything

Reply:
Rolled back the component, nothing to fix here